dev-python/pycpio/pycpio-1.3.2.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11..12} )
+inherit distutils-r1
+
+DESCRIPTION="Python CPIO library"
+HOMEPAGE="https://github.com/desultory/pycpio"
+SRC_URI="https://github.com/desultory/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+IUSE="test"
+RESTRICT="!test? ( test )"
```

Comment:
This is not required, it is added automatically by `distutils_enable_tests`

Reply:
In particular, you don't directly use `test` anywhere but instead it is inferred from distutils_enable_tests, so distutils_enable_tests handles everything internally which it needs to handle.

It will even handle the case where unittest doesn't require any dependencies at all, and thus, no IUSE or RESTRICT need to be defined. That only happens once you bump the minimum PYTHON_COMPAT to 3.12 (since unittest needs a special package on 3.11 to ensure it correctly fails when it detects 0 tests to run).